docs/guides/openshock-first-setup.md

```diff
@@ -18,15 +22,16 @@
 1. Connect your **Phone** to the hub:
     1. Plug your hub in and ensure it has power.
     2. On your phone, search for a Wi-Fi network named similar to ``OpenShock-XX:XX:XX:XX:XX:XX`` and connect to it.  
-    ??? Info "Image"
         ![Image "image"](../static/guides/first-setup/WiFioverview.png)
 2. Connect to the hub via the network:
     1. Open your phone's browser and type in ``10.10.10.10`` *or* ``openshock.local`` this should open up a web-interface for the hub.
     2. Find your router's Wi-Fi network in the web-interface.
     3. Press the green button next to it and type in your Wi-Fi password, then press submit.
     *A green pop up should appear if it's connected successfully*
-    ??? Info "Image"
         ![Image "image"](../static/guides/first-setup/ESPWebGUI.png)
+
+!!! warning "RF TX Pin" **DO NOT** change the RF TX Pin **UNLESS IT IS NOT AUTOMATICALLY DETECTED**. This is an advanced feature. It should be set correctly by default after flashing the OpenShock firmware. If the pin is not automatically selected (ie. the transmitter doesn't send any signals) you can reflash the board and input the correct pin in the web UI, or open a UART terminal and send the command `rftxpin #` where `#` is your pin number. For more information see the page dedicated to your micro-controller under [boards](../hardware/boards/index.md).
```

Comment:
There shouldnt be a need to reflash, just change the pin via serial or wifi

Reply:
You could just copy the comment you removed below and then explain after how to change the pin I guess